idl/duplication.thrift

```diff
@@ -66,6 +66,9 @@ struct duplication_add_request
     // - if false, duplication start state=DS_LOG,
     // server will replay and send plog mutation to follower cluster derectly
     3:optional bool is_duplicating_checkpoint = true;
+
+    // Specify the app name of remote cluster.
+    4:optional string remote_app_name;
```

Comment:
For the newly added fields, describe since which version (i.e., 2.6) they are added, as how this do in `struct app_info`.

Reply:
Is it safe to use app **name**, what happen if the app renaming in remote cluster? Or even swap app names in remote cluster?

Reply:
Good idea, I think it's reasonable to rename app of remote cluster while duplicating. We could support this on another pull request.